### PR TITLE
[PROGRAMMES-6693] If someone inputs an incorrect pid, don’t 500…

### PR DESCRIPTION
**Pull request by benolot**
Ticket: https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6693

Very simple fix for some "great" twig behavior. If something is null it will be true for "defined", as we use null as our fallback value, we need to add a specific check for that.

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: templates/base_ds2013.html.twig [hunk 1]
```diff
@@ -117,7 +117,7 @@
             <div id="programmes-content" class="visually-hidden" tabindex="-1">Main content</div>
             {% block body %}{% endblock %}
         </div>
-        {% if programme is defined %}{{ ds2013('footer', programme) }}{% endif %}
+        {% if programme is defined and programme is not null %}{{ ds2013('footer', programme) }}{% endif %}
```
Comment (BjornTwachtmann) [inline templates/base_ds2013.html.twig]: Could this not just if `if programme`

Comment (benolot) [reply templates/base_ds2013.html.twig]: No because if it's not defined that will error

Comment (BjornTwachtmann) [reply templates/base_ds2013.html.twig]: ffs twig

**Pull request merged**